test/fgpu/test_engine.py

```diff
@@ -933,6 +939,61 @@ async def test_missing_heaps(
             == np.sum(~dst_present) * n_substreams
         )
 
+        # Sensor is not present in the narrowband mode.
+        if output.decimation == 1:
+            # Information needed to check that the dig-rms-dbfs sensor is correctly
+            # going into FAILURE when there are missing samples.
```

Comment:
This comment seems a bit misplaced now (probably my fault when I hacked this approach together).

Reply:
I went through it to try and make sure everything was sensible but this is obviously one that I missed. Removed.